eng/build.ps1

```diff
@@ -581,6 +581,9 @@ function Deploy-VsixViaTool() {
   # Disable background download UI to avoid toasts
   &$vsRegEdit set "$vsDir" $hive HKCU "FeatureFlags\Setup\BackgroundDownload" Value dword 0
 
+  # Disable text spell checker to avoid spurious warnings in the error list
+  &$vsRegEdit set "$vsDir" $hive HKCU "FeatureFlags\Editor\EnableSpellChecker" Value dword 0
```

Comment:
should check with the editor to see if they plan to keep this flag around or will be enabling by default and removing the opt-out (or maybe there will be a different opt-out switch)

to avoid being broken later if they decide to change this